aws-datastore/src/androidTest/java/com/amplifyframework/datastore/AWSDataStorePluginInstrumentedTest.java

```diff
@@ -46,6 +47,7 @@
  * which were defined by the schema in:
  * testmodels/src/main/java/com/amplifyframework/testmodels/commentsblog/schema.graphql.
  */
+@Ignore("Currently does not pass.") // TODO: fix the error; re-enable test
```

Comment:
Can you move this to just the specific `@Test` that's failing? Having this at the class level will prevent some well-functioning tests from being exercised.

Reply:
When i tried just ignoring the `blogOwnerSavedIntoDataStoreIsThenQueriableInRemoteAppSyncApi`, the other test also failed. I think this test failing initially crashes the whole test suite, which made us unable to catch it last time.

Reply:
Hm, alright. We can come back to it.